Docs/Configuration.tex

```diff
@@ -2193,15 +2176,15 @@ \subsection{Force Properties}\label{kernelpropsforce}
   \textbf{Failsafe}: Empty\\
   \textbf{Description}: Adds kernel driver on specified macOS version or older.
 
-  \emph{Note}: Refer to \hyperlink{kernmatch}{\texttt{Add} \texttt{Add MaxKernel} description} for matching logic.
+  \emph{Note}: Refer to \hyperlink{kernmatch}{\texttt{Add} \texttt{MaxKernel} description} for matching logic.
```

Comment:
Can't this just be `\texttt{Add\ MaxKernel}`?

Reply:
Keeping consistent with other examples.

Reply:
On reflection, decided to just change all examples